pageserver/src/tenant/storage_layer/layer.rs

```diff
@@ -1897,7 +1898,7 @@ impl ResidentLayer {
         self.owner.metadata()
     }
 
-    #[cfg(test)]
+    /// Cast the layer to a delta, return an error if it is indeed an image layer.
```

Comment:
```suggestion
    /// Cast the layer to a delta, return an error if it is an image layer.
```

Reply:
not a native speaker of english but I think the use of `indeed` here is a bit strange.